pipelines/smartseq2_single_sample/SS2SingleSample.wdl

```diff
@@ -0,0 +1,136 @@
+##### Naming Rules #####
+## ** Files should be named in CamelCase starting with a capital
+## ** Tasks should be named in CamelCase starting with a capital
+## ** Alias' should be named in CamelCase starting with a capital
+## ** Variable declarations should be lower_snake_case
+
+##### New Line Rules #####
+## ** New line before and after `calls`
+## ** New Line between task definitions
+## ** New lines between blocks in tasks
+## ** New line variable declarations that you want to group
+## ** New line between imports and workflow definition
+## ** No new lines at the start or end of any workflow/call/task definition block
+## ** No need for having two new lines anywhere
+## ** No new lines at the start of any blocks
+
+##### Spacing Rules #####
+## ** Space before `{` characters
+## ** Space after `,` characters
+## ** Space before and after operators e.g. `+`, `=`
+## ** Space after `:` character in runtime block
+## ** Space after `#` when creating a comment (or `##` if we decide thats our comment definer)
+
+##### Tab vs Space for Indents ######
+## ** Always use spaces? I think whats most frequently used is whenever you want to indent you use 2 spaces but we should decide what works best for us.
+
+##### Runtime Attributes ######
+## ** This will change quite a bit i believe when the new spec of WDL comes out (WDL SPEC 3) but we should define all values we care about
+## ** even if they are the default values.  e.g. write `cpu: 1` instead of leaving it out because its the default.
+
+##### Comment Rules ######
+## **  Should comments have one or two `##` before a comment.  Maybe two for workflow or task descriptions and one for any other kind of comment?
+
+import "Hisat2QcPipeline.wdl" as RunHisatQCWDL  ## ** Should import alias follow some kind of pattern like ending in "WDL" or something else
+import "Hisat2RsemPipeline.wdl" as RunHisatRSEMWDL
+
+## ** Explanations above the workflow about what inputs it expects, what outputs/metrics it outputs
```

Comment:
This sort of information would be better included in meta and parameter_meta blocks

Reply:
I always forget about the meta/parameter_meta blocks!

Reply:
The below is not very clear from me from the documentation, which has no examples. Can you link me a WDL that uses this? It sounds useful and I'd like for us to utilize it. 

```
Parameter Metadata Section
🐖 Cromwell supported ✅

$parameter_meta = 'parameter_meta' $ws* '{' ($ws* $parameter_meta_kv $ws*)* '}'
$parameter_meta_kv = $identifier $ws* '=' $ws* $string
This purely optional section contains key/value pairs where the keys are names of parameters and the values are string descriptions for those parameters.

Additional requirement: Any key in this section MUST correspond to a parameter in the command line

Metadata Section
🐖 Cromwell supported ✅

$meta = 'meta' $ws* '{' ($ws* $meta_kv $ws*)* '}'
$meta_kv = $identifier $ws* '=' $ws* $string
This purely optional section contains key/value pairs for any additional meta data that should be stored with the task. For example, perhaps author or contact email.
```

Reply:
here is an example of  it in a task - https://github.com/openwdl/wdl/blob/master/versions/draft-3/SPEC.md#example-3-runtimemetadata , it would look similar at the workflow level as well